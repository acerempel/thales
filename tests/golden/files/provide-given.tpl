Carrots are orange
{for adjective in load-template("provide-given/adjectives.tpl", {shape = "conical", potato = 1}).adjectives}  and {adjective}
{end}
