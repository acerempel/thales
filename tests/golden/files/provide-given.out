Carrots are orange
  and nice
  and conical
  and sometimes purple

